src/install.ts

```diff
@@ -116,6 +118,21 @@ function installSbt() {
   core.endGroup();
 }
 
+function installCoursier() {
+  core.startGroup("Install Coursier");
+  curl(
+    "https://git.io/coursier-cli-linux",
```

Comment:
This command will fail when running on macOS and Windows. I was pretty sure the CI was configured to test these cases but the CI didn't seem to run 🤔